src/org/zaproxy/zap/extension/selenium/resources/help/contents/intro.html

```diff
@@ -44,6 +50,10 @@
 	</ul>
 
 	<p>
+		<strong>Firefox/geckodriver Note:</strong> There's an issue (<a
+			href="https://bugzilla.mozilla.org/show_bug.cgi?id=1103196">Bug 1103196</a>) that
+		prevents HTTPS sites from being used in versions &ge; 48 and &lt; 53.
+	<p>
 		<strong>PhantomJS Note:</strong> There's an issue (<a
 			href="https://github.com/ariya/phantomjs/issues/11342">Issue #11342</a>) that prevents
 		sites at localhost, 127.0.0.1 and ::1 from being proxied through ZAP. Until a fix is
```

Comment:
Unrelated, but since it's here and I'm reading it 😀 , you can work around this issue my manually setting up a hosts entry? (i.e.: www.mytestsite.com 127.0.0.1)